migrations/educandu-2023-04-04-01-rename-accreditedEditors-to-allowedEditors-in-documents.js

```diff
@@ -0,0 +1,17 @@
+export default class Educandu_2023_04_04_01_rename_accreditedEditors_to_allowedEditors_in_documents {
```

Comment:
The date is wrong here now 😄

Reply:
Not anymore :stuck_out_tongue_closed_eyes: